### PR TITLE
docs fixes

### LINKED ISSUE #2165: Doc site issues
### Describe what should be investigated or refactored
- The air gap course is available, we should update https://docs.zarf.dev/docs/getting-started/understand-the-basics#airgap-basics 
- https://docs.zarf.dev/docs/the-zarf-cli/cli-commands/zarf_tools_helm_dependency 2.2.0 of what tool? Helm? Should we mention that helm is built into zarf anyway? 
- https://docs.zarf.dev/docs/create-a-zarf-package/zarf-components#files files section reads a bit awkwardly, it says two things it can be and also that it can be verified. Seems like the verified part should be separate
- https://docs.zarf.dev/docs/create-a-zarf-package/zarf-components?file-examples=Local&chart-examples=URL+%28oci%29&manifest-examples=Remote&import-examples=Local+Path#component-imports OCI url doesn’t have an example. It's just blank when you click it/
- https://docs.zarf.dev/docs/deploy-a-zarf-package/deployment-ui#connect-to-existing-cluster Maybe we should say "deploy into an existing cluster" rather than "connect to an existing cluster"
- https://docs.zarf.dev/docs/deploy-a-zarf-package/deployment-ui#additional-package-commands issue with image
- https://docs.zarf.dev/docs/deploy-a-zarf-package/k9s-dashboard we have [ but never close the brace

### PR DESCRIPTION
**Pull request by AustinAbro321**
## Description
docs fixes

## Related Issue

Fixes #2165 

## Type of change
- [X] Other (security config, docs update, etc)

### CODE REVIEW

File: docs/1-getting-started/0-understand-the-basics.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Before you can effectively use Zarf, it is useful to have an understanding of th
 ### AirGap Basics
 
 - [What is AirGap](https://ibm.github.io/kubernetes-networking/vpc/airgap/)
-- AirGap Kubernetes Course - Coming Soon!
+- [AirGap Kubernetes Course](https://training.linuxfoundation.org/training/modern-air-gap-software-delivery-lfs281/) - Created by Defense Unicorns
```
Comment (AustinAbro321) [inline docs/1-getting-started/0-understand-the-basics.md:19]: I put in a Created by Defense Unicorn Note here cause I thought we may want to people to know that. Not sure though - thoughts @Racer159

File: docs/3-create-a-zarf-package/2-zarf-components.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Files can be:
 
 - Relative paths to either a file or directory (from the `zarf.yaml` file)
 - A remote URL (http/https)
-- Verified using the `shasum` field for data integrity (optional and only available for files)
+  - Verified using the `shasum` field for data integrity (optional and only available for files)
```
Comment (Racer159) [inline docs/3-create-a-zarf-package/2-zarf-components.md:44]: There seems to be an added space here.

File: docs/4-deploy-a-zarf-package/3-deployment-ui.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ Once a package is deployed into the cluster, the Web UI offers additional comman
 - Update: Use when you wish to update a package with a new version of the same package.
 - Remove: Use when you wish to remove a package and all of its resources from the cluster. This cannot be undone.
 
-![Web UI deployed packages table with a context menu showing additional package commands](../.images/dashboard/Web%20UI%20-%20package%20commands.png
+![Web UI deployed packages table with a context menu showing additional package commands](../images/dashboard/Web%20UI%20-%20package%20commands.png
```
Comment (Racer159) [inline docs/4-deploy-a-zarf-package/3-deployment-ui.md:93]:
```suggestion
![Web UI deployed packages table with a context menu showing additional package commands](../.images/dashboard/Web%20UI%20-%20package%20commands.png)
```
The issue is that htis doesn't have a trailing `)`

**Review by Racer159 — CHANGES_REQUESTED** [review]

File: docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ appending '/index.yaml' to the URL, it should be able to retrieve the chart
 repository's index. Note: 'repository' can be an alias. The alias must start
 with 'alias:' or '@'.
 
-Starting from 2.2.0, repository can be defined as the path to the directory of
+A repository can be defined as the path to the directory of
```
Comment (AustinAbro321) [inline docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md:41]: This is helm 2.2.0 right? We're bringing in helm anyway and 2.2.0 was released 7 years ago so it seems unnecessary to add this. This one is just my opinion though. Does the users version of helm ever matter when their using zarf?

Comment (Racer159) [reply docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md:41]: We can edit this message for Zarf though this whole file originally came from the Helm project hence why it is geared towards Helm a bit more - TLDR, Helm currently has their cmd stuff in their main package and it isn't importable (even with //go:linkname)

File: docs/1-getting-started/0-understand-the-basics.md (hunk 1 above)
Comment (Racer159) [reply docs/1-getting-started/0-understand-the-basics.md:19]: I think that is fine, but @andrewg-xyz / @TheFutonEng are their any attribution requirements from LF?

File: docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md (hunk 4 above)
Comment (AustinAbro321) [reply docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md:41]: Ah in that case maybe we say `starting from helm 2.2.0`, I just felt it read a bit awkwardly

Comment (AustinAbro321) [reply docs/2-the-zarf-cli/100-cli-commands/zarf_tools_helm_dependency.md:41]: Oh it's actually copied by `make docs-and-schema ` anyway. Probably not a big enough deal to change

File: docs/1-getting-started/0-understand-the-basics.md (hunk 1 above)
Comment (AustinAbro321) [reply docs/1-getting-started/0-understand-the-basics.md:19]: If they have an issue with it we will PR it back per convo

**Review by Racer159 — APPROVED** [review]
lgtm!

**Pull request merged**